redact/redact.go

```diff
@@ -0,0 +1,59 @@
+package redact
+
+import (
+	"net/http"
+	"strings"
+)
+
+var SensitiveFields = []string{
+	"authorization",
+	"cookie",
+	"set-cookie",
+
+	"access_token",
+	"refresh_token",
+	"id_token",
+	"accesstoken",
+	"refreshtoken",
+	"idtoken",
+
+	"password",
+	"secret",
+}
+
+func IsSensitive(key string) bool {
+	for _, sensitiveKey := range SensitiveFields {
+		if strings.Contains(strings.ToLower(key), sensitiveKey) {
```

Comment:
Since we allow the list to be modified, and particularly working with headers it may feel natural to have them capitalized.
```suggestion
		if strings.Contains(strings.ToLower(key), strings.ToLower(sensitiveKey)) {
```